### DISCUSSION TITLE
🖋: 前提知識の書き方は変えたほうが良い？

### DISCUSSION THREAD

**Discussion by takei0w0** [ws-4020/mobile-app-crib-notes#128] [Ideas]
学習コンテンツFBより

- リンク先のボリュームが大きい
- ToDoアプリ実装中にリンクがあると嬉しい

より範囲を絞るべき？前提知識のリンクをToDoアプリ実装に混ぜ込んだほうがいい？

**Comment by tani-takanori** [comment]
前提知識を読んだらどうなるかを書いてあげればいいのでは？
 - ToDoアプリの実装例や説明の理解ができるようになる.
 - 用語やコード例など、わからないことがあるときに参照してもらう

**Comment by creature-water-valley** [comment]
>ToDoアプリの実装例や説明の理解ができるようになる.
用語やコード例など、わからないことがあるときに参照してもらう

これに対しては、以下の文言を追加済みなのでクローズする。
>ToDoアプリの実装例や解説を理解するために参考となるサイトを紹介します。 用語やコード例でわからないことがあるときは参照してください。

[前提知識](https://ws-4020.github.io/mobile-app-crib-notes/react-native/pre-requisites)
